rcljava_common/include/rcljava_common/exceptions.hpp

```diff
@@ -19,6 +19,55 @@
 
 #include "rcljava_common/visibility_control.hpp"
 
+/// Execute \a error_statement if an exception has happened.
+/**
+ * \param env a JNIEnv pointer, used to check for exceptions.
+ * \param error_statement statement executed if an exception has happened.
+ */
+#define RCLJAVA_COMMON_EXCEPTION_CHECK_X(env, error_statement) \
```

Comment:
sounds reasonable to me

Reply:
The only problem with `RCLJAVA_COMMON_CHECK_FOR_EXCEPTION_AND_RETURN` is that it sounds like it always returns ...
I think I will use some variadics macro magic so both can be named in the same way.

Reply:
Fixed in https://github.com/osrf/ros2_java/pull/30/commits/e78003d7fd41b3512711daf444990352703d8eea.
I have picked a slightly different name to avoid the issue mentioned above (and to above variadics macro magic).